modules/odc-interacting-with-applications-and-components.adoc

```diff
@@ -16,8 +16,16 @@ This feature is available only when you create applications using the *From Git*
 ====
 +
 * Hover your cursor over the lower left icon on the Pod to see the name of the latest build and its status. The status of the application build is indicated as *New* (image:odc_build_new.png[title="New Build"]), *Pending* (image:odc_build_pending.png[title="Pending Build"]), *Running* (image:odc_build_running.png[title="Running Build"]), *Completed* (image:odc_build_completed.png[title="Completed Build"]), *Failed* (image:odc_build_failed.png[title="Failed Build"]), and *Canceled* (image:odc_build_canceled.png[title="Canceled Build"]).
-* The status or phase of the pod is indicated by different colors and tooltips as *Running* (image:odc_pod_running.png[title="Pod Running"]), *Not Ready* (image:odc_pod_not_ready.png[title="Pod Not Ready"]), *Warning*(image:odc_pod_warning.png[title="Pod Warning"]), *Failed*(image:odc_pod_failed.png[title="Pod Failed"]), *Pending*(image:odc_pod_pending.png[title="Pod Pending"]), *Succeeded*(image:odc_pod_succeeded.png[title="Pod Succeeded"]), *Terminating*(image:odc_pod_terminating.png[title="Pod Terminating"]), or *Unknown*(image:odc_pod_unknown.png[title="Pod Unknown"]).
-For more information about pod status, see the link:https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-phase[Kubernetes documentation].
+* The status or phase of the pod is indicated by different colors and tooltips as:
```

Comment:
I see this is existing text. But should _or phase_ be set-off in commas? This makes it seem like _status_ and _phase_ are different things. (I assume they are the same thing.)

Reply:
They are the same thing. However, previously we didn't provide full descriptions of each phase. We only linked to the Kubernetes doc. There were 3 phases not documented as they are on the OCP side. So, a bulleted list made more sense.